CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## vNext (TBD)
 
+### Breaking changes
+* The generated parts are now named `.realm.dart` instead of `.g.dart`. This is because the builder is now a `PartBuilder`, instead of a `SharedPartBuilder`. This implies that other builders can consume the output.
```

Comment:
Is there any special setup users need to do to take advantage of this? Might be a good idea to add a minimal example of passing the Realm generator output to another builder.

Reply:
Also I think we need to be explicit about what people need to do to get their project up-and-running again - it may be kind of obvious, but it's usually better to have too much info than too little.

Reply:
Added suggestions